php/commands/rewrite.php

```diff
@@ -173,10 +173,22 @@ public function structure( $args, $assoc_args ) {
 	 * : Show rewrite rules from a particular source.
 	 *
 	 * [--fields=<fields>]
-	 * : Limit the output to specific fields. Defaults to match,query,source.
+	 * : Limit the output to specific fields.
+	 * ---
+	 * default: match,query,source
```

Comment:
Hm, I'm not quite sure how I feel about this. Can you think of a better way to handle multiple default values?